### PR TITLE
First experience as code reviewer

### PR DESCRIPTION
**Pull request by lmunozm1702**
Add the html, css and assets folders submitted on the Microverse trials.

This is the W2-D1 exercise.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+   <!-- <div class="bg-image" 
+     style="background-image: url('https://mdbootstrap.com/img/Photos/Others/images/76.jpg');
+            height: 100vh">
+-->
```
Comment (lmunozm1702) [inline index.html:15]: :pray: Please remember to delete all comments from your codebase. This is to get a clean code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      <input class="btn btn-primary mb-3" type="submit" id="submit">
+      </div>
+    </form>
+    </div>
+
+
+
+<!--  
+  </div>
+-->
```
Comment (lmunozm1702) [inline index.html:89]: :pray:  Please remember to delete all comments from your codebase. This is to get a clean code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+   <!-- <div class="bg-image" 
+     style="background-image: url('https://mdbootstrap.com/img/Photos/Others/images/76.jpg');
+            height: 100vh">
+-->
+    <div class="container-md bg-info bg-opacity-25 ">
+    <h1 id="title">This is the title</h1>
```
Comment (lmunozm1702) [inline index.html:17]: This paragraph should be into a <header> tag. This way will be semantically correct and the screen readers and other technologies will thank you. 👍

File: index.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
```
Comment (lmunozm1702) [inline index.html:18]: This section of code should be into a <main> tag (until the end of the form). This way will be semantically correct and the screen readers and other technologies will thank you. 👍

File: index.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+      <label for="radio1" id="radio1-label">Select an option</label>
+      <div class="mb-3 form-check">
+      
+      <input class="form-check-input" type="radio" name="radio1" id="radio1" value="value1">value1
+      </div>
+      <div class="mb-3 form-check">
+      <input class="form-check-input" type="radio" name="radio1" id="radio2" value="value2">value2
+      </div>
+      
+
```
Comment (lmunozm1702) [inline index.html:53]: :arrow_forward: Kindly deletes excessive blank lines in your code.

:eyes:  Remember the "Best practices" followed in Microverse:
**_Be careful with blank lines and indentation._**
- Do not add blank lines, spaces, or indentations without a reason.
- For readability, add blank lines to separate large or logical code blocks.
- For readability, add two spaces of indentation. Do not use the tab key.
- on some code editors, you can set the tab button to do an X amount of spaces.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (lmunozm1702) [inline README.md:5]: I think you can improve your README file. Please 🙏🏽 follow the instructions from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md). Kindly add a meaningful title to your pull request.

If you don't have yet the knowledge to complete some info into the template leave it blank and fill it out when you have it.

:reminder_ribbon: Remember to recognize the contribution of BABEL and freecodecamp guys by the CSS/js you used in your project.

**Review by lmunozm1702 — COMMENTED** [review]
the_office_so_close

Hi @lmunozm1702,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### :high_brightness: Highlights :high_brightness: 

Required Changes ♻️

 Please make sure to add the files in your repository to activate the linters used by Microverse to review your code.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @lmunozm1702 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉👏🥇✨

## Highlights

- [x] Good description ✔️
- [x] Nice use of inline comments ✔️
- [x] Polite language ✔️ 

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**